src/Psl/Result/Failure.php

```diff
@@ -18,6 +18,8 @@ final class Failure implements ResultInterface
 {
     /**
      * @psalm-var Te
+     *
+     * @readonly
```

Comment:
Is this possible?
I thought it was not, since the exception is mutable?

Cool :)